src/server/cea_608_encoder/byte_pair_generator.py

```diff
@@ -7,6 +11,21 @@
     'CEA-608'
 ]
 
+def save_json_to_file(caption_data: dict):
+    # datetime object containing current date and time
+    now = datetime.now()
+    # mm.dd.YY_H:M:S
+    dt_string = now.strftime("%m.%d.%Y_%H:%M:%S")
+
+    path = pathlib.Path(__file__).parent.parent.parent.parent
```

Comment:
Can you find a way to get the Perception directory without having to know how many parent levels above it is? If we move this file to a new directory, this will be hard to debug an error

Reply:
I've added a config.py file at the /src/ directory to store the path to the Perception folder.